app/code/core/Mage/Catalog/Model/Resource/Abstract.php

```diff
@@ -699,6 +699,45 @@ public function getAttributeRawValue($entityId, $attribute, $store)
         return $attributesData ? $attributesData : false;
     }
 
+    /**
+     * Retrieve attribute's raw text value from DB.
```

Comment:
Hi @justinbeaty, I wanted to test this but actually I couldn't understand (from a first read) what text value the function is returning, and maybe the docblock could be a little bit more verbose?

Cause at the end of the day "Retrieve attribute's raw text value from DB" it's the almost the same of the function name so it doesn't help me much understanding what the function is doing

:-)

Reply:
I updated the docblock to "Retrieve attribute's raw value from DB using its source model if available." Not sure if that's still good enough, so if there's any suggestions I'm all ears.